site/docs/components/tabs.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Tabs
+navTitle: Tabs (not built)
+summaryParagraph: Tabs organize content by grouping similar information into containers that are shown and hidden through navigation.
+tags: ["Tab list", "Tab group", "Tab panel", "Section tabs", "Nav bar", "Vertical tabs", "Horizontal tabs"]
+needToKnow:
+- Navigation tabs are always used for global navigation at the top of the page or inside an offscreen menu on small screens.
+- Section tabs are used for chunking content, allowing movement between content within a single section of a page.
+---
+
+import DosAndDonts from "docs-components/DosAndDonts"
+import Do from "docs-components/Do"
+import Dont from "docs-components/Dont"
+
+
+## Visuals
+
+### UI Kit
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D18760%253A108" allowfullscreen></iframe>
+
+## Options
+
+- Navigation tabs and Section tabs:
+    - Navigation tabs are always used for global navigation at the top of the page or inside an offscreen menu on small screens.
+    - Section tabs are used for chunking content, allowing movement between content within a single section of a page.
+- Vertical tabs and horizontal tabs are used for establishing hierarchy as appropriate for the product area.
+- Default and reversed
+
+## To keep in mind
+
+- Anatomy:
+    - Tab list: This refers to the container holding each of the tab items.
+    - Selected tab: The current, active tab. One tab—and only one tab—is always selected for a given tab list.
+    - Unselected tab: The other available tabs.
+    - Tab item:
+        - A tab item contains a large clickable target area, a label, a selected tab indicator (when the tab is selected), and optional metadata.
+        - Tab link: The large clickable area of the tab item uses a link to navigate to that tab's content.
+        - Tab label: The label describes what content will be displayed. Use 1 short noun or noun phrase.
+        - Selected tab indicator: This contains a selected tab indicator to show that the tab is current.
+        - Optional metadata: You might display 1 item of metadata, such as a [Badge](/components/badge) to bring data forward from the content in that section or a [Tag](/components/tag) to add supplementary information, such as "New".
+    - Tab content: The content that lives "inside" a tab and is revealed when that tab item is selected.
+    - Separator: To separate tabs from content, use whitespace or the edge of another component, such as Title Blocks or Cards, not a [Divider](/components/divider).
```

Comment:
Reviewing various examples for our needs, it seems more likely that we always want to use a solid line such as a divider NOT just whitespace.

Reply:
Oh interesting, I thought we were trending in the other direction.

Reply:
We were, though we're seeing more examples for our needs where the solid line creates a better sense of grouping, ensuring the tabs are perceived as closely related and completely separate from any nearby elements. For example, some tabs might appear on the left when there are also action buttons on the right.